configuration/cardano/shelley-dummy-genesis.json

```diff
@@ -0,0 +1,41 @@
+{
+    "activeSlotsCoeff": 0.99,
+    "protocolMagicId": 838299499,
+    "systemStart": "2020-01-01T00:20:40Z",
+    "genDelegs": {},
+    "updateQuorum": 12,
+    "maxMajorPV": 25446,
+    "maxLovelaceSupply": 100000000,
+    "initialFunds": {},
+    "networkMagic": 403,
+    "networkId": "Testnet",
+    "epochLength": 21600,
+    "securityParam": 2160,
+    "slotLength": 20,
```

Comment:
I thought we decided to use 1s?

Reply:
The "dummy" in the name means it actually is a dummy file.

It satisfies the requirements:

* It exists
* It parses

Nothing more.